cloud/services/routetables/routetables.go

```diff
@@ -41,8 +40,27 @@ func (s *Service) Reconcile(ctx context.Context, spec interface{}) error {
 	if !ok {
 		return errors.New("invalid Route Table Specification")
 	}
+
+	existingRouteTable, err := s.Get(ctx, s.Scope.ResourceGroup(), routeTableSpec.Name)
+	if !azure.ResourceNotFound(err) {
+		if err != nil {
+			return errors.Wrapf(err, "failed to get route table %s in %s", routeTableSpec.Name, s.Scope.ResourceGroup())
+		}
+
+		// route table already exists
+		// currently don't support:
+		//  1. creating separate control plane and node (#718) so update both
+		//  2. can not specify your own routes via spec
```

Comment:
nit: this is a double negation
```suggestion
		//  2. specifying your own routes via spec
```